webpack.config.js

```diff
@@ -76,7 +76,7 @@ export default {
   mode: isProduction ? 'production' : 'development',
   entry: {
     index: [
-      fileURLToPath(new URL('web_src/js/jquery.js', import.meta.url)),
+      fileURLToPath(new URL('web_src/js/globals.js', import.meta.url)),
```

Comment:
Here, the old jquery.js should be removed?

Reply:
yes indeed, i forgot to `git rm` it.